tools/tool-testing/selftest.js

```diff
@@ -1605,11 +1611,19 @@ var getFilteredTests = function (options) {
   if (! files.inCheckout()) {
     tagsToSkip.push('checkout');
   }
-  if (options.offline) {
-    tagsToSkip.push('net');
-  }
-  if (! options.includeSlowTests) {
-    tagsToSkip.push('slow');
+  if (options.galaxyOnly) {
+    tagsToSkip.push('non-galaxy');
+  } else {
+    tagsToSkip.push('galaxy');
+    if (options.offline) {
+      tagsToSkip.push('net');
+    }
+    if (! options.includeSlowTests) {
```

Comment:
Yeah, `--galaxy` implies both `---slow` & `--online`. 

I guess, actually, there could be some sort of `galaxyTestTags` export in `galaxyUtils` and then this wouldn't have to happen, but I actually think that it is a fine interface to do it this way.